### ISSUE TITLE
Support for AlmaLinux / Rocky Linux

### ISSUE THREAD

**Issue by mattgresham** [CrowdStrike/aws-ssm-distributor#65]
A couple of our vendors have based their AMI's on AlmaLinux or Rocky Linux. They should be binary compatible so shouldn't need additional testing.

**Comment by ffalor** [comment]
Hey @mattgresham how do these instances show up in Ssm? Does running the current distributor package so the os is unsupported?

**Comment by mattgresham** [comment]
Yep, they show up in SSM and in the first step (GetSSMInstanceInformation) they are detected with these outputs

```
...
  "AgentVersion": "3.2.923.0",
  "IsLatestVersion": false,
  "PlatformType": "Linux",
  "PlatformName": "AlmaLinux",
  "PlatformVersion": "8.8",
...
```

The distributor then continues to step 5 (ExecuteDistributorPackageLinux) and errors with the following when it doesn't know which binary to use in the manifest:

```
failed to find platform: no manifest found for platform: almalinux, version 8.8, architecture x86_64
```

**Comment by ffalor** [comment]
@mattgresham I've been working on this, and started to test, but looks like almalinux & rockly linux are[ not supported by distributor](https://docs.aws.amazon.com/systems-manager/latest/userguide/distributor.html#:~:text=platforms%20and%20architectures.-,Supported%20package%20platforms%20and%20architectures,-You%20can%20use). 

```
[ERROR] Failed to build and publish new distributor package version in region us-east-1. Error An error occurred (InvalidDocumentContent) when calling the CreateDocument operation: Unsupported Platform included in manifest: almalinux.
```

**Comment by ffalor** [comment]
Closing this as this is a limitation of AWS Systems Manager, if they add support I will reopen this.

**Comment by ffalor** [comment]
@mattgresham it looks like aws now supports these, ill look into implementing them.

**Comment by mattgresham** [comment]
Thanks for following-up!

**Comment by ffalor** [comment]
Hey @mattgresham version 2.1.0 of the distributor package now supports these operating systems.

**Comment by mattgresham** [comment]
Thanks for letting me know!
